benchmarks/dynamo/torchbench.py

```diff
@@ -99,6 +100,9 @@ def setup_torchbench_cwd():
     "pyhpc_turbulent_kinetic_energy",
     "maml",
     "llama",
+    "doctr_det_predictor",
+    "doctr_reco_predictor",
+    "cm3leon_generate",
```

Comment:
This applies to both cpu and cuda? Consider to add reasons?

Reply:
Same reason with original `SKIP_TRAIN` models, there are no training implementation for those torchbench models for now

Reply:
Yes, these models not designed for training now